app/models/order.rb

```diff
@@ -3,6 +3,7 @@ class Order < ApplicationRecord
   has_many :order_freelancers
   has_many :freelancers, through: :order_freelancers
 
+  enum status: ["Ordered", "Paid", "Cancelled", "Completed"]
```

Comment:
The enum decision here was good -- we'll need to see if there's any residual trouble with the capitalized truthy enum methods.